less/svg.less

```diff
@@ -30,10 +30,13 @@
   }
 
   &.is-svg-paused .svg__playpause {
-    opacity: 0;
     .icon-video-play;
   }
 
+  &.is-svg-paused.hide-controls .svg__playpause {
+    visibility: hidden;
```

Comment:
I meant more in the sense that `opacity:0` was a way of not using `visibility:hidden` which prevents focus.

Reply:
So, this new `hide-controls` class is meant to completely hide it after using the Visua11y "no animations" option. If SVG used JSX templates, we could just remove it via a model property change like we do for Graphic Lottie. 

Speaking of, I'd like to change the templates to JSX once this ticket is closed/merged. Could refactor it at that point?

Reply:
Cool, and yessss.